src/js/view/controls/controls.js

```diff
@@ -302,52 +303,63 @@ export default class Controls extends Events {
                     break;
                 case 13: // enter
                 case 32: // space
-                    api.playToggle(reasonInteraction());
+                    if (document.activeElement.className === 'jw-shortcuts-switch' && evt.keyCode === 32) {
+                        // Let event bubble up so the spacebar can control the toggle if focused on
+                        return true;
+                    }
+                    if (shortcutsEnabled) {
+                        api.playToggle(reasonInteraction());
+                    }
                     break;
-                case 37: // left-arrow, if not adMode and settings menu is hidden
-                    if (!adMode && menuHidden) {
+                case 37: // left-arrow, if shortcuts are enabled, not adMode, and settings menu is hidden
+                    if (shortcutsEnabled && !adMode && menuHidden) {
```

Comment:
We should look into a way of only having to check shortcutsEnabled a single time, and adapting the case handlers based on the value a single time. There are a few good patterns for doing this, let's touch base later and we can pick one.